### PR TITLE
Draft eslint improvements

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

Attempt to standardise eslint config to use @guardian packages.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,98 +1,99 @@
...
+		"@guardian/tsconfig": "^0.2.0",
+		"@rollup/plugin-babel": "^6.0.3",
+		"@rollup/plugin-commonjs": "^25.0.0",
+		"@rollup/plugin-html": "^1.0.2",
+		"@rollup/plugin-node-resolve": "^15.1.0",
+		"@rollup/plugin-replace": "^5.0.2",
+		"@rollup/plugin-strip": "^3.0.2",
+		"@rollup/plugin-terser": "^0.4.3",
+		"@rollup/plugin-typescript": "^11.1.1",
+		"@semantic-release/github": "^9.0.2",
```
Comment (joecowton1) [inline package.json:62]:
```suggestion
		"@semantic-release/github": "^9.0.3",
```

File: package.json [hunk 2]
```diff
@@ -1,98 +1,99 @@
...
+		"@types/jest": "^29.5.2",
+		"@types/mime": "^3.0.1",
+		"@types/node": "^20.2.5",
```
Comment (joecowton1) [inline package.json:65]:
```suggestion
		"@types/node": "^20.3.0",
```

File: package.json [hunk 3]
```diff
@@ -1,98 +1,99 @@
...
+		"ci:monitoring": "yarn run node-riffraff-artifact"
+	},
+	"prettier": "@guardian/prettier",
+	"devDependencies": {
+		"@babel/core": "^7.22.1",
+		"@babel/plugin-proposal-class-properties": "^7",
+		"@babel/plugin-transform-runtime": "~7.22.4",
+		"@babel/preset-env": "^7.22.4",
+		"@babel/preset-typescript": "^7.21.5",
+		"@babel/runtime": "^7.22.3",
```
Comment (joecowton1) [inline package.json:47]:
```suggestion
		"@babel/runtime": "^7.22.5",
```

File: package.json [hunk 4]
```diff
@@ -1,98 +1,99 @@
...
+		"prepare": "husky install",
+		"ci:monitoring": "yarn run node-riffraff-artifact"
+	},
+	"prettier": "@guardian/prettier",
+	"devDependencies": {
+		"@babel/core": "^7.22.1",
+		"@babel/plugin-proposal-class-properties": "^7",
+		"@babel/plugin-transform-runtime": "~7.22.4",
+		"@babel/preset-env": "^7.22.4",
+		"@babel/preset-typescript": "^7.21.5",
```
Comment (joecowton1) [inline package.json:46]:
```suggestion
		"@babel/preset-typescript": "^7.22.5",
```

File: package.json [hunk 5]
```diff
@@ -1,98 +1,99 @@
...
+		"validate": "npm-run-all tsc test lint build",
+		"prepare": "husky install",
+		"ci:monitoring": "yarn run node-riffraff-artifact"
+	},
+	"prettier": "@guardian/prettier",
+	"devDependencies": {
+		"@babel/core": "^7.22.1",
+		"@babel/plugin-proposal-class-properties": "^7",
+		"@babel/plugin-transform-runtime": "~7.22.4",
+		"@babel/preset-env": "^7.22.4",
```
Comment (joecowton1) [inline package.json:45]:
```suggestion
		"@babel/preset-env": "^7.22.5",
```

File: package.json [hunk 6]
```diff
@@ -1,98 +1,99 @@
...
+		"test:unit": "jest --coverage",
+		"tsc": "tsc",
+		"types": "tsc --project ./tsconfig.build.json",
+		"validate": "npm-run-all tsc test lint build",
+		"prepare": "husky install",
+		"ci:monitoring": "yarn run node-riffraff-artifact"
+	},
+	"prettier": "@guardian/prettier",
+	"devDependencies": {
+		"@babel/core": "^7.22.1",
```
Comment (joecowton1) [inline package.json:42]:
```suggestion
		"@babel/core": "^7.22.5",
```

**Comment by akinsola-guardian** [comment]
This was included into this PR

https://github.com/guardian/consent-management-platform/pull/845

**Pull request closed without merging**